arrays-intermediate.js

```diff
@@ -17,10 +21,14 @@ function sumArray(numbers) {
 **************************************************************/
 function findFirstStringStartingWithLetter(letter, strings) {
   //TODO: Add your code here
+  strings = strings.map((value, index, array) => {
```

Comment:
You didn't have to loop over every string in the array to make them lower cased, instead you could've just done that inside the filter method like so:
`` strings.filter( (value)=> value.toLowerCase().startsWith(letter))``